### PR TITLE
doc/dev/cephadm.rst: clarify, fix spelling nits

### PR DESCRIPTION
**Pull request by dmick**
Signed-off-by: Dan Mick <dmick@redhat.com>

## Checklist
- [ ] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/dev/cephadm.rst [hunk 1]
```diff
@@ -61,18 +62,18 @@ To start a test cluster::
...
 part of the system, though, can you get away with smaller changes so that
 cpatch runs faster.  For instance::
 
-  sudo ../src/script/cpach -t quay.io/ceph-ci/ceph:8f509f4e --py
+  sudo ../src/script/cpatch -t quay.io/ceph-ci/ceph:8f509f4e --py
 
 will update the mgr modules (minus the dashboard).  Or::
 
-  sudo ../src/script/cpach -t quay.io/ceph-ci/ceph:8f509f4e --core
+  sudo ../src/script/cpatch -t quay.io/ceph-ci/ceph:8f509f4e --core
```
Comment (mgfritch) [inline doc/dev/cephadm.rst:76]: @ideepika has a similar patch (#35653)

File: doc/dev/cephadm.rst [hunk 2]
```diff
@@ -89,14 +90,15 @@ When you're done, you can tear down the cluster with::
...
+mon command handler thread within the MGR, thus preventing any concurrent
+CLI calls. Note that pressing ``^C`` will not resolve this situation,
+as *only* the client will be aborted, but not execution of the command
+within the orchestrator manager module itself. This means, cephadm will
+be completely unresponsive until the execution of the CLI handler is
+fully completed. Note that even ``ceph orch ps`` will not respond while
+another handler is executing.
+
+This means we should do very few synchronous calls to remote hosts.
+As a guideline, cephadm should do at most ``O(1)`` network calls in CLI handlers.
```
Comment (mgfritch) [inline doc/dev/cephadm.rst:103]:
```suggestion
As a guideline, we should do very few synchronous calls to remote hosts meaning, 
cephadm should do at most ``O(1)`` network calls in CLI handlers.
```

Minor nit .. the term `this means` is used a lot in this section ..

**Comment by sebastian-philipp** [comment]
Oh sorry! Unfortunately github doesn't add a DCO to those commits. I guess you'll have to do a 

```
git pull
```
and then squash the commits.

**Comment by sebastian-philipp** [comment]
ping?

**Comment by dmick** [comment]
Sorry, missed any responses on this because I'm unable to work out how to filter emails from github to "what's relevant to me".

I'll have to investigate DCO; I thought it was signing the commits.

**Comment by dmick** [comment]
Oh, I see, someone added another commit to this PR and didn't sign it.  I think that makes the English worse, too.  I guess I'll suggest a better wording with a third commit, and if we agree, I'll squash before push.

**Comment by dmick** [comment]
no, on rereading, I really do like the original better.  The guideline (such as it is) is O(1) calls; the "avoiding synchronous calls" isn't a guideline, it's a requirement.  And in the revised version, "meaning..." is awkward and would need some rephrase.

I propose the original single commit.  Any objection?

**Comment by sebastian-philipp** [comment]
> I propose the original single commit. Any objection?

sounds good to me!

**Comment by dmick** [comment]
What do we do about the dashboard API tests?

**Comment by dmick** [comment]
retest this please

**Comment by dmick** [comment]
retest this please

**Comment by dmick** [comment]
retest this please

**Pull request merged**